docker-compose-examples/keycloak-mysql.yml

```diff
@@ -23,6 +23,7 @@ services:
         MYSQL_PASSWORD: password
         KEYCLOAK_USER: admin
         KEYCLOAK_PASSWORD: Pa55w0rd
+        JDBC_PARAMS: "connectTimeout=30000"
```

Comment:
Not sure we should have this in the example. Can you elaborate on why you added it?

Reply:
I think it would be useful include a JDBC parameter in the exemple, although I could not think in a better example than that. You think is better removing this parameter here?

Reply:
I would remove it here. This example is meant as a starting point to use MySQL and if there is no specific reason to add it then it may confuse people.

Perhaps instead add a separate yml file to show custom JDBC_PARAMS would be better? Something like keycloak-mysql-custom-params.yml?

Reply:
I thought it would have many duplicated information in a keycloak-mysql-custom-params.yml file. Instead I commented the JDBC_PARAMS variable with a warning message in 15799ebe527e93dfc3ab4df2b5290440048cd110
What do you think?